### PR TITLE
[MIG] disable_openerp_online > disable_odoo_online

### PR DESCRIPTION
**Pull request by tungocbui**

### CODE REVIEW

**Comment by max3903** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Please rename it to _disable_odoo_online_

File: disable_openerp_online/__openerp__.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 ##############################################################################
 {
     "name": "Remove odoo.com bindings",
-    "version": "8.0.1.1.0",
+    "version": "9.0.0.1.0",
```
Comment (pedrobaeza) [inline disable_openerp_online/__openerp__.py]: 9.0.1.0.0

File: disable_openerp_online/static/src/js/disable_openerp_online.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
-    // resulting in 'Your OpenERP is not supported'
-    instance.web.WebClient.include({
-        show_annoucement_bar: function() {}
-    });
-};
+odoo.define('disable_openerp_online.announcement', function (require) {
+	"use strict";
+	var WebClient = require('web.WebClient');
+	WebClient.include({
+	    show_announcement_bar: function() {
```
Comment (pedrobaeza) [inline disable_openerp_online/static/src/js/disable_openerp_online.js]: Is this function still valid in v9?

**Comment by tungocbui** [comment]
@pedrobaeza thks for reviewing the pull request. I just fixed the source code taken into your comments. About the function "show_announcement_bar", it's still valid in v9 (you can find it in addons/mail/static/src/js/announcement.js). Please let me know if you have any other comments.

File: disable_odoo_online/static/src/js/disable_openerp_online.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+odoo.define('disable_openerp_online.announcement', function (require) {
```
Comment (pedrobaeza) [inline disable_odoo_online/static/src/js/disable_openerp_online.js]: This should be `odoo.define('disable_odoo_online.announcement', ...`

Comment (tungocbui) [inline disable_odoo_online/static/src/js/disable_openerp_online.js]: Thks for pointing this out. I will check other places also.

File: disable_odoo_online/views/disable_openerp_online.xml [hunk 4]
```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <openerp>
     <data>
-        <template id="assets_backend" name="disable_openerp_online assets" inherit_id="web.assets_backend">
+        <!-- the name of template will affect the order include the js,
+        this js file should load after annoucement.js to override function
+        in  annoucement.js file -->
+        <template id="assets_backend" name="mail disable_openerp_online assets" inherit_id="web.assets_backend">
```
Comment (sebalix) [inline disable_odoo_online/views/disable_openerp_online.xml]: It should be `mail disable_odoo_online assets`.

File: disable_odoo_online/views/disable_openerp_online.xml [hunk 5]
```diff
@@ -1,7 +1,10 @@
...
             <xpath expr="." position="inside">
                 <script type="text/javascript" src="/disable_openerp_online/static/src/js/disable_openerp_online.js"></script>
```
Comment (sebalix) [inline disable_odoo_online/views/disable_openerp_online.xml]: `src="/disable_odoo_online/static/src/js/disable_odoo_online.js"`

File: disable_odoo_online/static/src/js/disable_openerp_online.js (hunk 3 above)
Comment (sebalix) [inline disable_odoo_online/static/src/js/disable_openerp_online.js]: The current file should be renamed to `disable_odoo_online.js`.

**Comment by sebalix** [comment]
Excepting minor fixes, :+1: thank you!

**Comment by tungocbui** [comment]
Thks @pedrobaeza @sebalix. I just committed the fixes for your comments. I also did the rename in po / pot files. I have 1 concern about the link to transifex (http://www.transifex.com/oca/OCA-server-tools-8-0/language/de) in po files, i don't know what should i do with it.

And yes, if you have any other comments, please let me know...

**Comment by pedrobaeza** [comment]
Some comments to finish:
- Remove the pot file. It's not needed anymore.
- In the file https://github.com/trobz/server-tools/blob/9.0-disable_openerp_online/disable_odoo_online/model/publisher_warranty_contract.py, please pass it to new API.

**Comment by tungocbui** [comment]
@pedrobaeza i just committed the fixes for
- Remove the pot file
- Port publisher_warranty_contract.py to new api format.

Have a nice day,
Tu

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**